web/src/shared/components/status/status.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { FC } from 'react';
+import { JobStatus } from 'api/typings/jobs';
 import { Text, Tooltip } from '@epam/loveship';
 import styles from './status.module.scss';
 
 type StatusProps = {
     color: string;
-    statusTitle: string;
+    statusTitle: JobStatus | string;
```

Comment:
Why we still need string?

Reply:
because the project has a lot of assumptions in TS, so if you remove the typing as a string at this point, then there will be a wave-like generation of errors throughout the project.